extensions/wikia/AdEngine/js/lookup/lookupFactory.js

```diff
@@ -2,10 +2,10 @@
 define('ext.wikia.adEngine.lookup.lookupFactory', [
 	'ext.wikia.adEngine.adContext',
 	'ext.wikia.adEngine.adTracker',
-	'ext.wikia.aRecoveryEngine.recovery.sourcePointHelper',
+	'ext.wikia.aRecoveryEngine.recovery.sourcePoint',
 	'wikia.lazyqueue',
 	'wikia.log'
-], function (adContext, adTracker, helper, lazyQueue, log) {
+], function (adContext, adTracker, recoveryHelper, lazyQueue, log) {
```

Comment:
What do you think of changing `recoveryHelper` in the functions arguments list to `sourcePoint` or `sourcePointRecovery`?